chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -53,7 +53,7 @@ metadata:
     release: {{ .Release.Name }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: {{ .Release.Service }}
-    executor: {{ .Values.executor }}
+    executor: {{ trunc -63 .Values.executor }}  # AWS ECS Executor name is too long.
```

Comment:
I would drop this. Truncating an executor import path will cause the executor to fail to load, but that failure will happen much later. So this kind of hides that. I think it's good for helm/k8s to catch the label being too long and then fixing that earlier.

Reply:
Pretty sure this label is just informational - it wouldn't break anything right? But still, we don't need it now.

Reply:
Okay, cool, that was the question I had here:

> Can someone clarify if the 50 char limit is a blocker for this PR? I see that the value is getting truncated which I think would completely break the import path, right? So as is, this PR is not mergeable?

So if the label thing means nothing (functionally), then we should keep the truncation on this line and also use the current/long module paths, since the newer short ones wont be available in the current images until a future release.

I'd still like to merge the short names for future use, since I like that they're shorter and tidier.

Does that sound good @jedcunningham @LipuFei ?

Reply:
I agree with whatever you agree. :)

By the way, I have removed the `trunc` for now. If you think it's better to leave it here, just let me know. :)

Reply:
I'm getting a little lost in the discussions... I'll try to summarize and see if you agree:

1. We will keep `trunc 63` because it's just an informational label on the Deployment.
2. We will add the long paths for the executors into the allowed values list to avoid the dependency on #39093 

Do you agree?